src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectAutoCRContainer.tsx

```diff
@@ -0,0 +1,141 @@
+import ProjectFormContainer from './ProjectForm';
+import { ProjectFormInput } from './ProjectForm';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../../utils/routes';
+import { getRequiredLinkTypeNames } from '../../../utils/link.utils';
+import ErrorPage from '../../ErrorPage';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import { CreateStandardChangeRequestPayload, useCreateStandardChangeRequest } from '../../../hooks/change-requests.hooks';
+import { ChangeRequestReason, ChangeRequestType, ProjectProposedChangesCreateArgs, WbsElementStatus } from 'shared';
+import * as yup from 'yup';
+
+const CreateProjectCRContainer: React.FC = () => {
+  const toast = useToast();
+  const history = useHistory();
+
+  const [projectManagerId, setProjectManagerId] = useState<string | undefined>();
+  const [projectLeadId, setProjectLeadId] = useState<string | undefined>();
+
+  const { mutateAsync, isLoading } = useCreateStandardChangeRequest();
+  const {
+    data: allLinkTypes,
+    isLoading: allLinkTypesIsLoading,
+    isError: allLinkTypesIsError,
+    error: allLinkTypesError
+  } = useAllLinkTypes();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (!allLinkTypes || allLinkTypesIsLoading) return <LoadingIndicator />;
+  if (allLinkTypesIsError) return <ErrorPage message={allLinkTypesError.message} />;
+
+  const requiredLinkTypeNames = getRequiredLinkTypeNames(allLinkTypes);
+
+  const defaultValues = {
+    name: String(),
+    budget: 0,
+    summary: String(),
+    teamId: String(),
+    carNumber: 0,
+    links: [],
+    crId: '',
+    goals: [],
+    features: [],
+    constraints: [],
+    rules: [],
+    projectLeadId,
+    projectManagerId
+  };
+
+  const schema = yup.object().shape({
+    name: yup.string().required('Name is required!'),
+    carNumber: yup.number().min(0).required('A car number is required!'),
+    teamId: yup.string().required('A Team Id is required'),
+    budget: yup.number().optional(),
+    summary: yup.string().required('Summary is required!'),
+    projectLeadId: yup.number().optional(),
+    projectManagerId: yup.number().optional(),
+    links: yup
+      .array()
+      .optional()
+      .of(
+        yup
+          .object()
+          .optional()
+          .shape({
+            linkTypeName: yup.string().optional(),
+            url: yup.string().optional().url('Invalid URL')
+          })
+      )
+  });
+
+  const onSubmit = async (data: ProjectFormInput) => {
+    const { name, budget, summary, links, teamId, carNumber } = data;
+
+    const rules = data.rules.map((rule) => rule.detail);
+    const goals = mapBulletsToStrings(data.goals);
+    const features = mapBulletsToStrings(data.features);
+    const otherConstraints = mapBulletsToStrings(data.constraints);
+
+    try {
+      const projectPayload: ProjectProposedChangesCreateArgs = {
+        name,
+        summary,
+        status: WbsElementStatus.Active,
+        teamIds: [teamId],
```

Comment:
why are you hardcoding this as just a single array

Reply:
The create project form doesn't have an option to include multiple teams, its justa  team dropdown where you select a single team.

Reply:
Lets update the form to allow multiple teams